src/5e-SRD-Backgrounds.json

```diff
@@ -0,0 +1,147 @@
+                },
+                {
+                    "index": "dwarvish",
+                    "name": "Dwarvish",
+                    "url": "/api/languages/dwarvish"
+                },
+                {
+                    "index": "elvish",
+                    "name": "Elvish",
+                    "url": "/api/languages/elvish"
+                },
+                {
+                    "index": "giant",
+                    "name": "Giant",
+                    "url": "/api/languages/giant"
+                },
+                {
+                    "index": "gnomish",
+                    "name": "Gnomish",
+                    "url": "/api/languages/gnomish"
+                },
+                {
+                    "index": "goblin",
+                    "name": "Goblin",
+                    "url": "/api/languages/goblin"
+                },
+                {
+                    "index": "halfling",
+                    "name": "Halfling",
+                    "url": "/api/languages/halfling"
+                },
+                {
+                    "index": "orc",
+                    "name": "Orc",
+                    "url": "/api/languages/orc"
+                },
+                {
+                    "index": "abyssal",
+                    "name": "Abyssal",
+                    "url": "/api/languages/abyssal"
+                },
+                {
+                    "index": "celestial",
+                    "name": "Celestial",
+                    "url": "/api/languages/celestial"
+                },
+                {
+                    "index": "draconic",
+                    "name": "Draconic",
+                    "url": "/api/languages/draconic"
+                },
+                {
+                    "index": "deep-speech",
+                    "name": "Deep Speech",
+                    "url": "/api/languages/deep-speech"
+                },
+                {
+                    "index": "infernal",
+                    "name": "Infernal",
+                    "url": "/api/languages/infernal"
+                },
+                {
+                    "index": "primordial",
+                    "name": "Primordial",
+                    "url": "/api/languages/primordial"
+                },
+                {
+                    "index": "sylvan",
+                    "name": "Sylvan",
+                    "url": "/api/languages/sylvan"
+                },
+                {
+                    "index": "undercommon",
+                    "name": "Undercommon",
+                    "url": "/api/languages/undercommon"
+                }
+            ]
+        },
+        "starting_equipment": "/api/starting-equipment/acolyte",
+        "feature": {
+            "index": "shelter-of-the-faithful",
+            "name": "Shelter of the Faithful",
+            "url": "/api/features/shelter-of-the-faithful"
+        },
+        "suggested_characteristics": {
```

Comment:
You can probably make use of the choice structure for these too.

Reply:
I could do. Mind you, AFAIK there's no specific limit to how many of each you can choose, so I may not be able to assign the `choose` value.

Reply:
In the "Customising a Background" section in the SRD, it says the following:
> Finally, choose two personality traits, one ideal, one bond, and one flaw

Reply:
But even if we don't use Choices for these, we should probably consider using arrays, rather than numerical keys like this.

Reply:
This was changed to use those exact values you mentioned. The numbers represent die values you would roll to get these characteristics if you wish to roll for them (as detailed in the SRD). The updated structure reflects this a bit better.